google-cloud-spanner/lib/google/cloud/spanner/admin/database/v1/doc/google/spanner/admin/database/v1/spanner_database_admin.rb

```diff
@@ -55,22 +66,22 @@ module State
           # @!attribute [rw] page_token
           #   @return [String]
           #     If non-empty, +page_token+ should contain a
-          #     Next_page_token from a
-          #     previous ListDatabasesResponse.
+          #     {Google::Spanner::Admin::Database::V1::ListDatabasesResponse#next_page_token Next_page_token} from a
```

Comment:
The link description has a case mistake. The method is `next_page_token`, not `Next_page_token`.

Reply:
Acknowledged. Filed https://github.com/googleapis/toolkit/issues/1568 to fix in the generator.